std_msgs/Header header
geometry_msgs/Vector3[] linear
geometry_msgs/Vector3[] angular
builtin_interfaces/Time[] t_k #desired execution of the trajectory
